docs/resources/segment.md

```diff
@@ -0,0 +1,54 @@
+# configcat_segment Resource
+
+Creates and manages a **Segment**.
```

Comment:
```suggestion
Creates and manages a **Segment**. Read more about segmentation and segments in ConfigCat [here].
```

Reply:
A többihez hasonlóan "What is a Segment.." lett a leírás.